backend/src/main/java/develup/api/DiscussionCommentApi.java

```diff
@@ -41,7 +42,11 @@ public ResponseEntity<ApiResponse<DiscussionCommentResponse>> createComment(
             @PathVariable Long discussionId,
             @RequestBody CreateDiscussionCommentRequest commentCreateRequest
     ) {
-        var response = discussionCommentService.createComment(accessor.id(), discussionId, commentCreateRequest);
+        DiscussionCommentResponse response = discussionCommentService.createComment(
+                accessor.id(),
+                discussionId,
+                commentCreateRequest
+        );
         return ResponseEntity.ok(new ApiResponse<>(response));
```

Comment:
[Request Changes]
```suggestion
        DiscussionCommentResponse response = discussionCommentService.createComment(
                accessor.id(),
                discussionId,
                commentCreateRequest
        );

        return ResponseEntity.ok(new ApiResponse<>(response));
```

리턴 전 한 줄 띄우는 것도 컨벤션이었던 걸로 알고 있습니다. 😊